examples/csharp/l2-resources/datadog-integration.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Security.Cryptography;
+using Amazon.CDK;
+using Constructs;
+using MongoDB.AWSCDKResourcesMongoDBAtlas;
+
+
+namespace CdkTestAppCsharp
+{
+    public class CdkTestAppCsharpStack : Stack
+    {
+        internal CdkTestAppCsharpStack(Construct scope, string id, IStackProps props = null) : base(scope, id, props)
+        {
+            apiKey = this.Node.TryGetContext("apiKey");
+            profile = this.Node.TryGetContext("profile");
+            projectId = this.Node.TryGetContext("projectId");
```

Comment:
[q] What happens if the value isn't present - does CDK fail gracefully with an appropriate error message?